packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/search/components/DoubleSeparator.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const DoubleSeparator = () => {
+  return <div className="double separator"></div>;
```

Comment:
`ui double separator`, are there accompanying less styles for this less component in this PR? :)

Reply:
There is not. I have not moved less components to nr-model, because I was not clear, If I register custom less components in nr-model, via entry point, to which template am I supposed to actually add this js/less entry point please?

Reply:
@mirekys The issue from my perspective, is that there are no html templates where to attach {{webpack[something]}} after you register it, as is the case in other libraries such as oarepo ui or vocabularies.

Reply:
by webpack[something] you mean custom-componetns.js/less?

Reply:
@mirekys I mean, after I create the components and do all the steps, I need to include this less (js) somewhere into a html template. Nr-model does not have any html templates. For example, in oarepo ui, we add this entry into javascript.html which is then inherited by nr-docs, but in nr-model we don't have templates where to include the less components so that it would actually be used in the project.